src/Facet/Carrier/Throw/Inject.hs

```diff
@@ -12,13 +12,13 @@ import Control.Carrier.Reader
 import Control.Effect.Throw
 import Control.Monad.IO.Class
 
-runThrow :: (f -> e) -> ThrowC e f m a -> m a
+runThrow :: (f -> m e) -> ThrowC e f m a -> m a
```

Comment:
This generalization enables us to lift unification errors into `Facet.Elab.Err` which is constructed in the monad.